reference/observability/fields-and-object-schemas.md

```diff
@@ -13,7 +13,7 @@ Beat modules (for example, [{{filebeat}} modules](beats://reference/filebeat/fil
 
 This reference covers:
 
-* [Logs Explorer fields](/reference/observability/fields-and-object-schemas/logs-app-fields.md)
+* [Logs Discover fields](/reference/observability/fields-and-object-schemas/logs-app-fields.md)
```

Comment:
The doc that lists the required fields for Logs Explorer is not really accurate for Logs Explorer and Discover (https://www.elastic.co/guide/en/observability/current/logs-app-fields.html). For example, host name, kubernetes pod id, etc are not required. @LucaWintergerst should we remove that page altogether?